src/linear/logistic_regression.rs

```diff
@@ -715,7 +715,7 @@ mod tests {
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
     #[test]
     fn lr_fit_predict() {
-        let x = DenseMatrix::from_2d_array(&[
+        let x: DenseMatrix<f64> = DenseMatrix::from_2d_array(&[
```

Comment:
using f64 instead of f32 for better precision in wasm32 tests

Reply:
probably f64 is more stable but possibly less supported